platform/source/mbed_application.c

```diff
@@ -1,5 +1,5 @@
 /* mbed Microcontroller Library
- * Copyright (c) 2017-2017 ARM Limited
+ * Copyright (c) 2017 ARM Limited
```

Comment:
2017-2019

Reply:
I simply corrected the copyright notice year. Does it warrant a copyright year update?

Reply:
You are right. 2017 is fine.